APP-MANAGER

```diff
@@ -14,6 +14,13 @@ CLI=$(basename "$0")
 arch="$HOSTTYPE"
 currentuser="$USER"
 
+# Colors
+RED='\033[0;31m'; LightBlue='\033[1;34m'; Green='\033[0;32m'
+
+web_get() {
```

Comment:
This is a bash script, this line should start with "function"

Reply:
I aware of this. Some bash require it some versions not
Not really sure which, but better to use that everywhere